restlight-core/src/main/java/io/esastack/restlight/core/resolver/reqentity/FlexibleRequestEntityResolverFactory.java

```diff
@@ -132,7 +131,7 @@ public Result<?, Void> readFrom(RequestEntity entity, RequestContext context)
             return checkRequired(nav, converter, Result.err());
         }
 
-        protected MediaType getMediaType(RequestEntity entity, HttpRequest request) {
+        protected MediaType getMediaType(RequestEntity entity, RequestContext requestContext) {
```

Comment:
It is recommended to unify the naming style. Using `ctx` to represent the `RequestContext` will be more concise and unified instead of `requestContext` which is a little lengthy